source/_components/climate.myicomfort.markdown

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
I was wondering about this item since the new platform has not been merged into HA yet.  What is the proper value for this key in this situation?  Thanks for your assistance.

Reply:
you can put the 0.87 release on it, if it appears that it will be a later release than we update it before we run a merge.